src/observable/dom/AjaxObservable.ts

```diff
@@ -67,7 +67,7 @@ export interface AjaxCreationMethod {
   post(url: string, body?: any, headers?: Object): Observable<AjaxResponse>;
   put(url: string, body?: any, headers?: Object): Observable<AjaxResponse>;
   delete(url: string, headers?: Object): Observable<AjaxResponse>;
-  getJSON<T, R>(url: string, headers?: Object): Observable<R>;
+  getJSON<R>(url: string, headers?: Object): Observable<R>;
```

Comment:
@blesh okay. I fixed in https://github.com/ReactiveX/rxjs/pull/2069/commits/501bac49bab8949a64f6b76a47ac47ec6b1bfec1